cli/azd/pkg/input/console.go

```diff
@@ -498,11 +539,122 @@ func (c *AskerConsole) Prompt(ctx context.Context, options ConsoleOptions) (stri
 	return response, nil
 }
 
+// Prompts the user for a single value
+func (c *AskerConsole) PromptDir(ctx context.Context, options ConsoleOptions) (string, error) {
```

Comment:
I'm guessing we'll need to validate the return string to be a valid directory.. From an external prompt, it's fatal, but for a CLI prompt, we may want to do a prompt loop?